Src/GoogleApis.Auth.DotNet4/OAuth2/ServiceAccountCredential.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/*
-Copyright 2013 Google Inc
+Copyright 2015 Google Inc

```

Comment:
You don't need to change the year in case you are changing a file.
You only add 2015 to new files :)